app/javascript/gobierto_data/webapp/components/sets/SQLEditorHeader.vue

```diff
@@ -252,8 +253,14 @@ export default {
 
     this.datasetId = this.$route.params.numberId
     this.token = getToken()
+    this.userId = getUserId()
+
+    this.noLogin = this.userId === "" ? true : false
   },
   methods: {
+    goToLogin() {
+      location.href='/user/sessions/new?open_modal=true'
```

Comment:
Why don't you use vue-router to handle this?

Reply:
This is using an old method from Gobierto,  this could be migrated to vue router but might take more time.